composer.json

```
@@ -47,7 +47,7 @@
         "symfony/dependency-injection": "^5.3",
         "symfony/dotenv": "^5.3",
         "symfony/expression-language": "^5.3",
-        "symfony/flex": "^1.17",
+        "symfony/flex": "^1.17|^2",
```

Tiny nitpick, let's use the same syntax as above. :-)

```suggestion
        "symfony/flex": "^1.17 || ^2.0",
```

---

Doesn't matter to me, I use the single one as it is what is used by Symfony.